modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -76,6 +76,24 @@ module FindPoaByParticipantId
               key: 'return'
             )
         end
+
+        module FindAssignedFlashes
+          DEFINITION =
+            Action.new(
+              service: ClaimantWebService::DEFINITION,
+              name: 'findAssignedFlashes',
+              key: 'return'
+            )
+        end
+
+        module AddFlash
+          DEFINITION =
+            Action.new(
+              service: ClaimantWebService::DEFINITION,
```

Comment:
Should it be `ClaimantService` here or `ClaimantWebService` in [`bgs_service`](https://github.com/department-of-veterans-affairs/vets-api/pull/19202/files#diff-74da4c3565160579b7a99a01fd4f4036371f3f169d50b8f2da823af48af2f364)?

Reply:
[See below](https://github.com/department-of-veterans-affairs/vets-api/pull/19202#discussion_r1838623808)

Glad to discuss if we think naming this after the latter part so it's a 1:1 or closer to. If nothing else, this conversation could lend to defining said consistency moving forward. We'll probably rework a good bit of this in to the old gem style structure ([bgs-ext](https://github.com/department-of-veterans-affairs/bgs-ext/tree/master) for reference) when we go to LHDI for it to be shared across our APIs.

Reply:
As in the below [thread](https://github.com/department-of-veterans-affairs/vets-api/pull/19202/files/b61774647a8de9eab1a1f2ec14616e55c45fb60b#r1838638221), I think `ClaimantService` is fine for this PR. Does that mean we should call it `ClaimantService::DEFINITION` here?

Reply:
I would move it to it's own ClaimantService space and chance the DEFINITION service to respective Service